Assets/Scripts/UI/UIScrollers/HorizontalScroller.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+using UnityEditor;
+using System.IO;
+
+public class HorizontalScroller : MonoBehaviour
```

Comment:
No namespace or class summary. (Explain what this class does)

Should be `namespace DN`.